bigtable/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 [1]: https://pypi.org/project/google-cloud-bigtable/#history
 
+## 0.31.0
```

Comment:
Given no feature / dependency changes, this would logically be a `0.30.3` release.  I presume you want to bump it to `0.31.0` to mark as `beta`?  In that case, we need to note it (maybe under `New Features`?)

Reply:
I'll note it in features.

Reply:
I've already noted it in "New Features"